src/arviz_plots/plots/ppcpitplot.py

```diff
@@ -47,12 +48,17 @@ def plot_ppc_pit(
 
     This plot shows the empirical cumulative distribution function (ECDF) of the PIT values.
     To make the plot easier to interpret, we plot the Δ-ECDF, that is, the difference between
-    the expected CDF from the observed ECDF. Simultaneous confidence bands are computed using
+    the observed ECDF and the expected CDF. Simultaneous confidence bands are computed using
     the method described in described in [1]_.
 
     Alternatively, we can visualize the coverage of the central posterior credible intervals by
     setting ``coverage=True``. This allows us to assess whether the credible intervals includes
-    the observed values. We can obtain the coverage of the central intervals from the PIT by
+    the observed values.
+
+    If the difference is positive, the model is under-confident: the predictions have a wider spread than the data -- they are too uncertain.
+    If the difference is negative, the model is over-confident: the predictions have a narrower spread than the data -- they are too certain.
+
+    We can obtain the coverage of the central intervals from the PIT by
```

Comment:
Better to link here https://arviz-devs.github.io/EABM/Chapters/Prior_posterior_predictive_checks.html#pit-ecdfs. The link should also be added to `plot_ecdf_pit` and `plot_loo_pit`